src/Redux/actionCreators/ServiceActions.js

```diff
@@ -29,6 +30,13 @@ export const fetchServiceSuccess = res => dispatch => {
       total_count: res.data.total_count,
     },
   });
+  if (res.data.total_count > 0) {
```

Comment:
@Vivek205 @chetankothari please review if this piece is fine. Setting up the strategy at the point of fetching the services